Sapodilla.java

```diff
@@ -1,8 +1,20 @@
 package om;
 
 public class Sapodilla {
+  private double weight;
   public String toString() {
   return "This is a Sapodilla created by Om";
-}
-}
+  }
+  
+  // Getter
+  public double getWeight() {
+  return weight;
+  }
+  
+  // Setter
+  public void setWeight (double newWeight) {
+  this.weight = newWeight;

```

Comment:
several indentation issues in this file, but will approve for sake of this assignment

Reply:
I am not getting it, can you please tell me specific lines, so that I can improve from next time?